### PR TITLE
Fixed FAQ: could not find checked attribute for "yes"

### PR DESCRIPTION
**Pull request by jeroendesloovere**
## Problem

On Safari, "yes" was never true.

### CODE REVIEW

File: src/Frontend/Modules/Faq/Js/Faq.js [hunk 1]
```diff
@@ -22,17 +23,19 @@ jsFrontend.faq.feedback =
 		// useful status has been changed
 		$('#usefulY, #usefulN').on('click', function()
 		{
-			// get useful status
-			var useful = ($('#usefulY').attr('checked') ? true : false);
+			// init useful status
+			var useful = false;
```
Comment (WouterSioen) [inline src/Frontend/Modules/Faq/Js/Faq.js]: can't you just do `var useful = $("#faqFeedbackForm input[type='radio']:checked").val() === 'Y';`?

**Comment by jeroendesloovere** [comment]
:+1:

**Comment by WouterSioen** [comment]
@jeroendesloovere can you fix merge conflicts?

**Comment by jeroendesloovere** [comment]
Created a new #1512

**Pull request closed without merging**